### DISCUSSION TITLE
Weekly Governance Update (11 March 2022)

### DISCUSSION THREAD

**Discussion by kaitlin-beegle** [filecoin-project/FIPs#320] [Announcements]
*This weekly report provides a high-level overview of what's happening with Filecoin governance.* 

*Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to participate.*

*If you have questions or ideas related to FIPs or Filecoin governance, please start a new discusison in the FIPs Forum or post directly in [#fil-fips](filecoinproject.slack.com).*

---
**Last Call FIPs** - final opportunity for community feedback! 
* N/A

**Current FIP Drafts** - *now accepting community feedback*
* FIP0027 - Change DealProposal label field from string to byte array [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0027.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/262)
* FIP0029 - Beneficiary address for storage providers [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/253)
* FIP0030 - Introducing the Filecoin Virtual Machine [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0030.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/287)
* FIP0031 - Atomic switch to non-programmable FVM [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0031.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/296)
* FIP0032 - Gas model adjustment for non-programmable FVM [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0032.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
* FIP0033 - Explicit premium for FIL+ Verified deals [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0033.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/243)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*
* FIP gas model adjustment for user programmability [(PR)](https://github.com/filecoin-project/FIPs/pull/317)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
* Set pre-commit deposit independent of sector content [(PR)](https://github.com/filecoin-project/FIPs/pull/307)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/290)
* Supporting contracts as deal clients in the built-in storage market actor [(PR)](https://github.com/filecoin-project/FIPs/pull/305)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/271)
* FIP Filecoin Message Replay Protection [(PR)](https://github.com/filecoin-project/FIPs/pull/300)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
* First draft of fungible token standard (following ERC-20) [(PR)](https://github.com/filecoin-project/FIPs/pull/280)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/277)
* Askv2 Protocol [(PR)](https://github.com/filecoin-project/FIPs/pull/224)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/223)

**🔥 HOT 🔥 Discussion Topics**
* [Change type of label in DealProposal to be []byte rather than string](https://github.com/filecoin-project/FIPs/discussions/262) - This proposal (FIP0027; see above) has been open for a while, and is now being newly reviewed. Pending the forthcoming FVM, network implementers have proposed changing the label type to a union type to better support the introduction of canonical Rust actors. 
* [Deep-Dive: Filecoin Improvement Proposals](https://github.com/filecoin-project/FIPs/discussions/314) - This discussion topic provides new documentation for the FIPs process and workflow.  It also codifies changes and set standards for FIP submissions, which were first trialed in the lead-up to the network v15 upgrade.  Community feedback strongly requested! 
* [FVM -driven gas model changes](https://github.com/filecoin-project/FIPs/discussions/316) - This is the first in a set of FIPs that will change the Filecoin network gas model to better support the forthcoming FVM. A great place to ask questions about how the introduction of the FVM may impact gas and other network fees. 

**Other Governance Updates**
* N/A - have a lovely weekend! ☀️

**Comment by Fatman13** [comment]
Great format!
